### PR TITLE
Environment variable check added

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
#57 [ Improvement ] [ Add a configuration validator in server.ts ]

File changed:
- [x] Notifier-Service/server.ts
- [x] Search-Service/server.ts

### CODE REVIEW

File: Notifier-service/src/server.ts [hunk 1]
```diff
@@ -16,9 +18,14 @@ dotenv.config()
...
 
 
-let app = new App();
-app.init()
-app.start((port) => {
-    console.log("Listening on port :"+port);
-})
+// Ensure required SERVER_PORT PORT have datatype of number
+if(isNaN(Number(process.env.SERVER_PORT))) {
+    throw new Error('The SERVER_PORT must be a number!')
```
Comment (rajatkb) [inline Notifier-service/src/server.ts]: * If throwing error then using if-else is not required.
* The exception will take case of safe guarding the code for app.

Comment (Kajol-Kumari) [reply Notifier-service/src/server.ts]: So, should i remove the if, else condition and only leave the line "isNaN(Number(process.env.SERVER_PORT)" ?

File: Notifier-service/package.json [hunk 2]
```diff
@@ -12,7 +12,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/dotenv-safe": "^8.1.0",
     "dotenv": "^8.2.0",
```
Comment (rajatkb) [inline Notifier-service/package.json]: * remove dot-env unless its a dependency for dot-env safe.

File: Search-Service/package.json [hunk 3]
```diff
@@ -12,7 +12,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/dotenv-safe": "^8.1.0",
     "dotenv": "^8.2.0",
```
Comment (rajatkb) [inline Search-Service/package.json]: * same as above.

File: Notifier-service/package.json (hunk 2 above)
Comment (rajatkb) [reply Notifier-service/package.json]: I will look into this one.

**Pull request merged**